dotnet/src/Connectors/Connectors.AssemblyAI/AssemblyAIAudioToTextExecutionSettings.cs

```diff
@@ -25,6 +42,20 @@ public TimeSpan PollingInterval
         }
     }
 
+    /// <summary>
+    /// How long to wait until the timeout exception thrown. Defaults to infinite.
+    /// </summary>
+    [JsonPropertyName("polling_timeout")]
+    public TimeSpan? PollingTimeout
+    {
+        get => this._pollingTimeout;
+        set
+        {
+            this.ThrowIfFrozen();
+            this._pollingTimeout = value;
+        }
+    }
+
     /// <inheritdoc/>
     public override PromptExecutionSettings Clone()
```

Comment:
Add new properties to clone method also.

Reply:
Should I deep clone `TranscriptParams`?

Reply:
Yep, avoid any reference changes, on the original and cloned.

Reply:
Done